client/app/components/packages/landuse-form/show.hbs

```diff
@@ -1209,34 +1208,307 @@
       <section class="form-section">
         <h2 class="section-header">
           <span id="proposed-site-characteristics" class="section-anchor"></span>
-          Proposed Site Characteristics [place holder]
+          Proposed Site Characteristic and Conditions
         </h2>
+      {{#each
+        (filter-by 'isDeleted' false @package.landuseForm.landuseGeographies)
+      as |landuseGeography|}}
```

Comment:
Is there a specific reason you included `isDeleted` here? 

If this is the `show` page then everything should already be saved to CRM. `isDeleted` is used as a "flag" so that ember knows to delete it when a user clicks "save". But once that user hits save, the record should be deleted for good, so I don't think we'd be dealing with that `isDeleted` "flag" anymore. 

https://guides.emberjs.com/v2.14.0/models/creating-updating-and-deleting-records/#toc_deleting-records

Reply:
My mistake.  I did not question the logic when copying from the edit to the show page.  I'll update.